pkg/middlewares/google_compute_instance_group_manager_expander.go

```diff
@@ -0,0 +1,47 @@
+package middlewares
+
+import (
+	"github.com/snyk/driftctl/pkg/resource"
+	"github.com/snyk/driftctl/pkg/resource/google"
+)
+
+type GoogleComputeInstanceGroupManagerExpander struct{}
+
+func NewGoogleComputeInstanceGroupManagerExpander() *GoogleComputeInstanceGroupManagerExpander {
```

Comment:
Please add a comment here to explain the goal of that middleware and why we are doing that.
And BTW this is not an expander, we should probably choose another naming.